pydantic_ai_slim/pydantic_ai/agent/__init__.py

```diff
@@ -249,6 +253,7 @@ def __init__(
             model_settings: Optional model request settings to use for this agent's runs, by default.
             retries: The default number of retries to allow for tool calls and output validation, before raising an error.
                 For model request retries, see the [HTTP Request Retries](../retries.md) documentation.
+            validation_context: Additional [validation context](https://docs.pydantic.dev/latest/concepts/validators/#validation-context) used to validate all outputs.
```

Comment:
Ah here's the link! Let's explicitly call it `Pydantic validation context`, instead of "Additional"